contracts/test/GrantRound.test.ts

```diff
@@ -71,23 +68,28 @@ describe('GrantRound', function () {
     await mockERC20.connect(mpUser).approve(roundContract.address, MaxUint256);
     await mockERC20.connect(donor).approve(roundContract.address, MaxUint256);
     await mockERC20.connect(payoutAdmin).approve(roundContract.address, MaxUint256);
-  });
+
+    return { mockERC20, roundContract };
+  }
 
   describe('Initialization', () => {
     it('deploys properly', async function () {
+      const { roundContract } = await loadFixture(setup);
```

Comment:
I think best practice for fixtures would be to avoid assigning variables outside of the fixture's scope within the fixture (e.g. like we do with `endTime` and `startTime` on L54-55), and instead load the fixture in a before each hook that assigns all the variables there. 

Plus that way you also don't need to call `loadFixture` in each function. I know the Waffle docs show it that way, but Uniswap does it the other way, e.g. [here](https://github.com/Uniswap/uniswap-v3-core/blob/b2c5555d696428c40c4b236069b3528b2317f3c1/test/UniswapV3Factory.spec.ts#L29-L41). I'm not sure what they need the `createFixtureLoader` method for. Would that be useful for us in any way?

Reply:
I will try this approach but my impression is that we will still need the global variables to access the contract object to make calls as well as signers.

Reply:
Yea they will still be global, just would be set in the before each hook instead of in the test itself

```javascript
let roundContract;

beforeEach(async () => { ({ roundContract }) = await loadFixture(setup) });
```

Reply:
Just dug into this and as written couldn't use this snippet. Also `createFixtureLoader` currently takes wallets and not signers unfortunately.